server/datasets/ingest.py

```diff
@@ -95,16 +99,19 @@ def _log(*args):
             msg = str(args[0])
         girder.logger.info(msg)
```

Comment:
I thought about it. I'm not a fan of the fact that the entire Job log gets stored on one continually-concatenated string, which could start to noticeable impact performance if we're ingesting the entire TCGA (even if we only add 1 high-level message per item to the Job log, it's tens of thousands of lines).

@zachmullen what are the prospects for improving the Job log, such that it's not reallocating new concatenated strings every time you push to it, and so it doesn't have to push the entire string to the server every time you update it? I'd propose storing the log as a list of strings, then do an `update({'$push':` call to store recently-added messages.